ignite/services/chain/serve.go

```diff
@@ -148,31 +150,29 @@ func (c *Chain) Serve(ctx context.Context, cacheStorage cache.Storage, options .
 					if c.served {
 						c.served = false
 
-						fmt.Fprintln(c.stdLog().out, "💿 Saving genesis state...")
+						c.ev.Send("💿 Saving genesis state...")
 
 						// If serve has been stopped, save the genesis state
 						if err := c.saveChainState(context.TODO(), commands); err != nil {
-							fmt.Fprint(c.stdLog().err, err.Error())
+							c.ev.Send(err.Error())
 							return err
 						}
 
 						genesisPath, err := c.exportedGenesisPath()
 						if err != nil {
-							fmt.Fprintln(c.stdLog().err, err.Error())
 							return err
 						}
-						fmt.Fprintf(c.stdLog().out, "💿 Genesis state saved in %s\n", genesisPath)
+						c.ev.Send(fmt.Sprintf("💿 Genesis state saved in %s", genesisPath))
```

Comment:
I think it is not a perfect plan. 
I see signature as
`func(b Bus) Sendf(format string, args ...interface{})`
It has variadic declaration in the end for placeholders. But `Send ` already has a variadic declaration for options.
The first idea to add options to the signature in the following way
`func(b Bus) Sendf(options []Option, format string, args ...interface{})`
But it seems not much better
Instead of 
```
n.ev.Send(
  fmt.Sprintf("Setting reward %s to the chain %d at height %d", coins.String(), launchID, lastRewardHeight),
  events.ProgressStarted(),
)
```

We got

```
n.ev.Sendf(
  []events.Options{events.ProgressStarted()},
 "Setting reward %s to the chain %d at height %d"
 coins.String(), launchID, lastRewardHeight),
)
```
Also if there are no options to pass it will be 
```
n.ev.Sendf([]events.Option{}, "Setting reward %s to the chain %d at height %d", coins.String(), launchID, lastRewardHeight)
```

Reply:
Implemented in 10bc126.

It is relevant to mention that `Sendf` function doesn't support event options in favor of having N args. The `Send` function should be used when event options need to be used as arguments.